hasura/metadata/databases/default/tables/tables.yaml

```diff
@@ -6,6 +6,7 @@
 - "!include public_claims.yaml"
 - "!include public_distributions.yaml"
 - "!include public_epoches.yaml"
+- "!include public_feedbacks.yaml"
```

Comment:
we could untrack this table since its not used in the app at all now

Reply:
Its being used in the scripts though, thats why I needed to add it (so that adminClient can use it)